### DISCUSSION TITLE
Public access request certificates

### DISCUSSION THREAD

**Discussion by Edelmeyder** [Keyfactor/ejbca-ce#108] [Q&A] [answered]
Hello! I have installed EJBCA and have configured the RA for public access as described [here](https://doc.primekey.com/ejbca790/ejbca-operations/ejbca-operations-guide/ra-operations-guide/configure-ejbca-for-public-access) but the behavior is not what I expected. 
I want to have the public access web where anyone can create a request for a certificate which will later on be reviewed and accepted or rejected by an admin. But what happens with the configuration described in the doc page is that anyone can make a "request" but it gets automatically accepted, i.e. the certificate gets created and issued instead of a request. Can anyone help me get the desired behavior? Thanks

**Comment by mike-agrenius-kushner** [comment, marked as answer]
Ah, we've got you covered, fam. 

Check out Approvals: https://doc.primekey.com/ejbca790/ejbca-operations/ejbca-ca-concept-guide/approvals

They give you super precise control over who gets to create/revoke/recover and end entity. 

Cheers,
Mike

**Reply by Edelmeyder** [reply]
Thank you! With that I got it working.
